src/middlewared/middlewared/plugins/jail.py

```diff
@@ -669,8 +669,23 @@ def fstab(self, jail, options):
         if verrors:
             raise verrors
 
-        _list = iocage.fstab(action, source, destination, fstype, fsoptions,
-                             dump, _pass, index=index)
+        try:
+            _list = iocage.fstab(
+                action, source, destination, fstype, fsoptions,
+                dump, _pass, index=index
+            )
+        except Exception as e:
```

Comment:
How can you be sure the exception raised has a `message` attribute?
Perhaps we should not be so broad on catching Exception.